mod-oai-pmh/src/test/resources/firebird/mod-configuration/reusable/update-configuration.feature

```diff
@@ -1,53 +1,50 @@
-Feature: Reset default OAIPMH configs
-
+Feature: update oai-pmh configuration settings
   Background:
-    * url baseUrl + '/configurations/entries'
+    * url baseUrl + '/oai-pmh/configuration-settings'
 
   @BehaviorConfig
   Scenario: set behavior config
     * copy template = behaviorTemplate
-    * set template.value = data
+    * set template.configValue = data
     Given path id
     And request template
     And header Accept = 'application/json'
     And header Content-Type = 'application/json'
     And header x-okapi-token = okapitoken
     And header x-okapi-tenant = testUser.tenant
     When method PUT
-    Then status 204
+    Then status 200
 
   @TechnicalConfig
   Scenario: set technical config
     * copy template = technicalTemplate
-    * set template.value = data
+    * set template.configValue = data
     Given path id
     And request template
     And header Accept = 'application/json'
     And header Content-Type = 'application/json'
     And header x-okapi-token = okapitoken
     And header x-okapi-tenant = testUser.tenant
     When method PUT
-    Then status 204
+    Then status 200
 
   @GeneralConfig
   Scenario: set general config
     * copy template = generalTemplate
-    * set template.value = data
+    * set template.configValue = data
     Given path id
     And request template
     And header Accept = 'application/json'
     And header Content-Type = 'application/json'
     And header x-okapi-token = okapitoken
     And header x-okapi-tenant = testUser.tenant
     When method PUT
-    Then status 204
+    Then status 200
```

Comment:
@kjain110 it was temporary change, please make change in mod-oai-pmh to return 204 for PUT request and revert here